quirks/password-rules.json

```diff
@@ -150,7 +150,7 @@
         "password-rules": "minlength: 8; maxlength: 12; required: lower; required: upper; required: digit;"
     },
     "citi.com": {
-        "password-rules": "minlength: 6; maxlength: 50; max-consecutive: 2; required: lower, upper; required: digit; allowed: [_!@$]"
+        "password-rules": "minlength: 8; maxlength: 64; max-consecutive: 2; required: lower, upper; required: digit; required: [~`!@#$%^&*()_-\/|];"
```

Comment:
This line should be:

```
"password-rules": "minlength: 8; maxlength: 64; max-consecutive: 2; required: lower, upper; required: digit; required: [-~`!@#$%^&*()_\/|];"
```

There's a subtle language quirk with character classes where, if `-` is present, it needs to be the first item in the class to take effect.

Reply:
Ok got it. I have made the updates